MHVLib/Clients/MHVPersonClient.m

```diff
@@ -0,0 +1,105 @@
+//
+//  MHVPersonClient.m
+//  MHVLib
+//
+//  Created by Nathan Malubay on 5/17/17.
+//  Copyright © 2017 Microsoft Corporation. All rights reserved.
+//
+
+#import "MHVPersonClient.h"
+#import "MHVConnectionProtocol.h"
+#import "MHVValidator.h"
+#import "MHVMethod.h"
+#import "MHVPersonInfo.h"
+#import "XSerializer.h"
+#import "MHVServiceResponse.h"
+#import "NSError+MHVError.h"
+
+@interface MHVPersonClient ()
+
+@property (nonatomic, weak) id<MHVConnectionProtocol> connection;
```

Comment:
This has to be weak because the connection has a reference to the person client. It would cause a retain cycle if the person client held a strong reference to the connection.